dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -418,6 +418,13 @@ TaskFunction createsMultiWidgetConstructPerfE2ETest() {
   ).run;
 }
 
+TaskFunction createListTextLayoutPerfE2ETest() {
+  return PerfTest.e2e(
```

Comment:
`PerfTest.e2e` accepts an `enableImpeller` flag if you want to eagerly add an Impeller version: https://github.com/flutter/flutter/blob/master/dev/devicelab/lib/tasks/perf_tests.dart#L874

Reply:
Done. This also mad eme realize one of the files was named `.dart.dart`... hah.